src/stories/Page.stories.tsx

```diff
@@ -34,164 +28,104 @@ const argObject = {
   },
   main: [
     <>
-      <Section>
-        <Header
-          title="Recent Blogs"
-          icon="BookOpenIcon"
-          actions={
-            <Actions>
-              <Action.Link url="https://google.com" text="DEV.to" />
-            </Actions>
-          }
-        />
-        <List>
-          {blogPosts.map((post) => (
-            <List.Item
+      <Block title="My writings" imagePath="https://avatars.githubusercontent.com/u/65030610?s=200&v=4" size={2} isEditable={true} actions={<Action.Link url="https://dev.to"/>}>
+        <Block.List>
+          {blogPosts.map((post, index) => (
+            <Block.List.Item
               type={post.type}
               title={post.title}
               additional={post.additional}
               subtitle={post.subtitle}
-              actions={<Action.Link url="https://google.com" text="Read article" />}
+              key={`Block-${index}`}
             />
           ))}
-        </List>
-      </Section>
-      <Section>
-        <Header
-          title="Open Source"
-          icon="CodeBracketSquareIcon"
-          actions={
-            <Actions>
-              <Action.Link url="https://github.com" text="GitHub profile" />
-            </Actions>
-          }
-        />
-        <Pills pills={["react.js", "vue.js", "angular.js", "ember.js", "svelte.js"]} />
-        <List>
-          {openSource.map((project) => (
-            <List.Item
+        </Block.List>
+      </Block>
+      <Block title="Open Source" imagePath="https://avatars.githubusercontent.com/u/9919?s=200&v=4" size={2} isEditable={true} actions={<Action.Link url="https://github.com"/>}>
+        <Block.Pills pills={["react.js", "vue.js", "angular.js", "ember.js", "svelte.js"]} />
+        <Block.List>
+          {openSource.map((project, index) => (
+            <Block.List.Item
               type={project.type}
               title={project.title}
               additional={project.additional}
               subtitle={project.subtitle}
               count={project.count && { value: project.count?.value, icon: project.count?.icon }}
+              key={`Block-${index}`}
             />
           ))}
-        </List>
-      </Section>
-      <Section>
-        <Header title="Web Projects" icon="GlobeAltIcon" />
-        <List>
-          {projects.map((project) => (
-            <List.Item
+        </Block.List>
+      </Block>
+      <Block imagePath="https://avatars.githubusercontent.com/u/24602613?s=200&v=4" size={2} isEditable={true} title="Deployed Projects">
+        <Block.List>
+          {projects.map((project, index) => (
+            <Block.List.Item
```

Comment:
@NilsJacobsen Like this API tho haha, but feels a bit limited e.g. Card or List could soon turn in giant components which have to handle a lot of logic. But I guess for the moment: cool!